bentoml/_internal/models/model.py

```diff
@@ -352,6 +392,36 @@ def from_dict(data: dict[str, str | dict[str, str]] | ModelContext) -> ModelCont
 
 @attr.frozen
 class ModelSignature:
+    """
+    A model signature represents a method on a model object that can be called.
+
+    Note that anywhere a ``ModelSignature`` is used, a ``dict`` with keys corresponding to the fields
+    can be used instead. For example, instead of ``{"predict": ModelSignature(batchable=True)}``, one
+    can pass ``{"predict": {"batchable": True}}``.
+
+    Fields:
```

Comment:
Should this also mention that batchable and batch_dim are a `bentoml.Runnable` concept, and this is used for converting models to Runnable/Runner in BentoML?